README.md

```diff
@@ -245,7 +245,7 @@ Take a look at the following overview of the Admin App extension samples we prov
 
 * [Custom DAM provider](./src/library-extender#custom-digital-assets-management-provider) - This extension is used to implement custom DAM providers. This sample demonstrates how to override one of the DAM providers which comes out of the box with the AdminApp.
 
-### Modifications done in the config.json
+### Modifications done in the Backend interface settings
```

Comment:
Shouldn't we remove this section's content and just add a link to the SF docs?

Reply:
Same for all child READMEs. I expect that this is all moved to SF docs and is not needed here.